features/account/src/main/kotlin/com/teacherworkout/features/account/auth/AuthenticateScreen.kt

```diff
@@ -0,0 +1,79 @@
+package com.teacherworkout.features.account.auth
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import com.teacherworkout.android.navigation.AppDestinations
+import com.teacherworkout.features.account.R
+import com.teacherworkout.features.account.composables.AccountScreenScaffold
+import com.teacherworkout.features.account.composables.EmailField
+import com.teacherworkout.features.account.composables.PasswordField
+import com.teacherworkout.features.account.composables.RegistrationLoadingUi
+import com.teacherworkout.features.account.composables.RequestFailedUi
+import org.koin.androidx.compose.getViewModel
+
+
+@Composable
+fun AuthenticateScreen(navController: NavHostController, viewModel: AuthViewModel = getViewModel()) {
+    val state by viewModel.state.collectAsState()
+
+    AccountScreenScaffold(titleId = R.string.auth_title, navController = navController) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(16.dp)
```

Comment:
It would help a lot to refactor all these dimensions into one file and reuse them from there. 
https://developer.android.com/jetpack/compose/resources#dimensions

Reply:
Yes, I should've done this from the start!